orchestra/utils/task_lifecycle.py

```diff
@@ -999,6 +1008,19 @@ def submit_task(task_id, task_data, snapshot_type, worker, work_time_seconds):
          'work_time_seconds': work_time_seconds
          })
 
+    # Temporarily map snapshot types onto iteration statuses
+    snapshot_type_iteration_status = {
+        TaskAssignment.SnapshotType.SUBMIT: Iteration.Status.REQUESTED_REVIEW,
+        TaskAssignment.SnapshotType.ACCEPT: Iteration.Status.REQUESTED_REVIEW,
```

Comment:
Nope—currently there's no difference in functionality between `SUBMIT` and `ACCEPT` snapshots!